dsc/workflows/base/simple_csv.py

```diff
@@ -95,7 +95,12 @@ def item_metadata_iter(
             for _, row in metadata_df.iterrows():
                 yield row.to_dict()
 
-    def prepare_batch(self) -> tuple[list, ...]:
+    def prepare_batch(
+        self,
+        *,
+        synced: bool = False,  # noqa: ARG002
```

Comment:
For workflows like `Wiley`, the `synced` boolean can be used to skip data retrieval steps if the data has been synced (e.g., in Prod, the batch "folder" and its contents will be _pulled_ from Stage).

**Also**, note that while `SimpleCSV` _does _not_ use the `synced` argument, the name is not leading with an underscore. This is because `mypy` will raise an error about the mismatched parameter names between the `@abstractmethod` and the implemented, and it's suspected that this is because the parameter is a keyword argument. 🤔 Ruff error `[ARG002](https://docs.astral.sh/ruff/rules/unused-method-argument/)` will need to be skipped as a result.

TLDR: Either `mypy` or Ruff would have an issue. Decided to ignore Ruff's error.